Bottom: 99c9c1e9913c91ea420dd1191ac82f4384363bc0
Top:    a5b644b48a1cd7f27f994b5ef01019827d25037f
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-12 11:12:25 -0500

Update docker-entrypoint to call parent container's entrypoint
    
    After updating the parent image, our entrypoint wasn't setting some environment
    variables correctly. Using the parent's entrypoint script after our setup
    fixes this problem.


---
diff --git a/docker/docker-mantisbt/docker-entrypoint.sh b/docker/docker-mantisbt/docker-entrypoint.sh
index c517f76a..2a948ac8 100755
--- a/docker/docker-mantisbt/docker-entrypoint.sh
+++ b/docker/docker-mantisbt/docker-entrypoint.sh
@@ -109,7 +109,7 @@ case ${1} in
         # Apache gets grumpy about PID files pre-existing
         rm -f /var/run/apache2/apache2.pid
         # Start apache
-        exec apache2 -D FOREGROUND
+        exec apache2-foreground
         ;;
 
     config_inc)
